### DISCUSSION TITLE
Npm installation error

### DISCUSSION THREAD

**Discussion by cmouli2502** [coreui/coreui-free-react-admin-template#386] [Q&A]
when installing npm install getting below error , tried with updating latest versions still same error 
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: @coreui/coreui-free-react-admin-template@4.3.1
npm ERR! Found: react@18.1.0
npm ERR! node_modules/react
npm ERR!   react@"^18.1.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^17" from @coreui/icons-react@2.0.1
npm ERR! node_modules/@coreui/icons-react
npm ERR!   @coreui/icons-react@"^2.0.1" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

**Comment by Pavitra554** [comment]
Check this article
https://www.codegrepper.com/code-examples/javascript/npm+ERR%21+ERESOLVE+unable+to+resolve+dependency+tree+npm+ERR%21+npm+ERR%21+While+resolving%3A+ViroSample%400.0.1+npm+ERR%21+Found%3A+react%4017.0.1
